tests/models/test_transformers.py

```diff
@@ -76,6 +76,23 @@ def test_transformer_smoke_select(model_name, model_kwargs):
     assert lm["answer"] in ["p", "t", "w"]
 
 
+@pytest.mark.needs_credentials
+@pytest.mark.skip("Need to figure out auth")
+def test_llama3_chat():
```

Comment:
Does this just need the `HF_TOKEN` (or whatever it is) environment variable set? If so, the `needs_credentials` mark should ensure it only runs on the build machines with that available. You can queue against the CI build to double check

Reply:
I'm not sure, I think you added this skip :) I just shuffled the tests around a bit to organize

Reply:
OK. Well, I can see about performing the above steps when things are slightly less frantic.